bff-service/src/app.service.ts

```diff
@@ -0,0 +1,73 @@
+import { CACHE_MANAGER, HttpException, HttpService, HttpStatus, Inject, Injectable } from '@nestjs/common';
+import { Cache } from 'cache-manager';
+
+@Injectable()
+export class AppService {
+  services;
+  cacheableUrls;
+
+  constructor(
+    @Inject(CACHE_MANAGER) private cacheManager: Cache,
+    private httpService: HttpService,
+  ) {
+    this.services = {
```

Comment:
Good code! But if we want add new service we need change code instead pass new .ENV variable